### PR TITLE
Release notes for 2.10.0

### PR DESCRIPTION
**Pull request by ViktarStarastsenka**
Please do not merge before Thursday.

### CODE REVIEW

File: content/ri/release-notes/v2.10.0.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+aliases: /ri/release-notes/v2.10.0/
+         /ri/release-notes/v2.10.0.md
+---
+## 2.10.0 (September 2022)
+This is the General Availability (GA) release of RedisInsight 2.10.0
+
+### Highlights
+- Formatters: Additional support for values of keys with Protobuf, Binary, PHP unserialize, and Java serialized objects, save selected formatters for other keys
+- Enhanced overview for Redis databases with [OSS Cluster API](https://docs.redis.com/latest/rs/clusters/optimize/oss-cluster-api/) enabled to see memory and key allocation, as well as monitor database information per shards
+- Configure Workbench to not clear the Editor after commands have been run and support for grouping the results
```
Comment (K-Jo) [inline content/ri/release-notes/v2.10.0.md:16]: Ability to configure the Workbench to not clear ...

File: content/ri/release-notes/v2.10.0.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- Launch of the user survey to collect feedback
```
Comment (K-Jo) [inline content/ri/release-notes/v2.10.0.md:17]: Launch of the user survey. (of course to collect feedback)

File: content/ri/release-notes/v2.10.0.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+- [#1017](https://github.com/RedisInsight/RedisInsight/pull/1017), [#1025](https://github.com/RedisInsight/RedisInsight/pull/1025), [#1029](https://github.com/RedisInsight/RedisInsight/pull/1029), [#1059](https://github.com/RedisInsight/RedisInsight/pull/1059), [#1092](https://github.com/RedisInsight/RedisInsight/pull/1092) Add support for additional data formats in Browser/Tree view, including Protobuf, Binary, Pickle, PHP unserialize, Java serialized objects
+- [#1130](https://github.com/RedisInsight/RedisInsight/pull/1130) Save formatters selected for other keys
+- [#1177](https://github.com/RedisInsight/RedisInsight/pull/1177) Add a validation when edited value is not valid in the selected format in Browser/Tree view
+- [#1048](https://github.com/RedisInsight/RedisInsight/pull/1048) Configure Workbench to not clear the Editor after commands have been run
+- [#1119](https://github.com/RedisInsight/RedisInsight/pull/1119) Move the Pipeline mode configuration for Workbench to the dedicated "Workbench" section in settings
+- [#1149](https://github.com/RedisInsight/RedisInsight/pull/1149) Group results in Workbench to see them in bulk
```
Comment (K-Jo) [inline content/ri/release-notes/v2.10.0.md:27]: when see them in bulk? What does it enable to the user?

File: content/ri/release-notes/v2.10.0.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+- Enhanced overview for Redis databases with [OSS Cluster API](https://docs.redis.com/latest/rs/clusters/optimize/oss-cluster-api/) enabled to see memory and key allocation, as well as monitor database information per shards
+- Configure Workbench to not clear the Editor after commands have been run and support for grouping the results
+- Launch of the user survey to collect feedback
+
+### Details
+**Features and improvements**
+- [#1159](https://github.com/RedisInsight/RedisInsight/pull/1159), [#1160](https://github.com/RedisInsight/RedisInsight/pull/1160), [#1068](https://github.com/RedisInsight/RedisInsight/pull/1068), [#1071](https://github.com/RedisInsight/RedisInsight/pull/1071), [#1095](https://github.com/RedisInsight/RedisInsight/pull/1095), [#1097](https://github.com/RedisInsight/RedisInsight/pull/1097), [#1098](https://github.com/RedisInsight/RedisInsight/pull/1098) Display enhanced overview for Redis databases with [OSS Cluster API](https://docs.redis.com/latest/rs/clusters/optimize/oss-cluster-api/) enabled to see memory and key allocation, as well as monitor database information per shards
```
Comment (K-Jo) [inline content/ri/release-notes/v2.10.0.md:21]: same same as above with/without

File: content/ri/release-notes/v2.10.0.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+Title: RedisInsight v2.10.0, September 2022
+linkTitle: v2.10.0 (September 2022)
+date: 2022-09-29 00:00:00 +0000
+description: RedisInsight v2.10.0
+weight: 1
+aliases: /ri/release-notes/v2.10.0/
+         /ri/release-notes/v2.10.0.md
+---
+## 2.10.0 (September 2022)
+This is the General Availability (GA) release of RedisInsight 2.10.0
```
Comment (K-Jo) [inline content/ri/release-notes/v2.10.0.md:11]: no need to have the .0 for patches.  check the templates @ViktarStarastsenka

File: content/ri/release-notes/v2.10.0.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+- [#1017](https://github.com/RedisInsight/RedisInsight/pull/1017), [#1025](https://github.com/RedisInsight/RedisInsight/pull/1025), [#1029](https://github.com/RedisInsight/RedisInsight/pull/1029), [#1059](https://github.com/RedisInsight/RedisInsight/pull/1059), [#1092](https://github.com/RedisInsight/RedisInsight/pull/1092) Add support for additional data formats in Browser/Tree view, including Protobuf, Binary, Pickle, PHP unserialize, Java serialized objects
+- [#1130](https://github.com/RedisInsight/RedisInsight/pull/1130) Save formatters selected for other keys
+- [#1177](https://github.com/RedisInsight/RedisInsight/pull/1177) Add a validation when edited value is not valid in the selected format in Browser/Tree view
+- [#1048](https://github.com/RedisInsight/RedisInsight/pull/1048) Configure Workbench to not clear the Editor after commands have been run
```
Comment (K-Jo) [inline content/ri/release-notes/v2.10.0.md:25]: Ability to configure the workbench ...

**Review by K-Jo — CHANGES_REQUESTED** [review]

File: content/ri/release-notes/v2.10.0.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+description: RedisInsight v2.10.0
+weight: 1
+aliases: /ri/release-notes/v2.10.0/
+         /ri/release-notes/v2.10.0.md
+---
+## 2.10.0 (September 2022)
+This is the General Availability (GA) release of RedisInsight 2.10.0
+
+### Highlights
+- Formatters: Additional support for values of keys with Protobuf, Binary, PHP unserialize, and Java serialized objects, save selected formatters for other keys
```
[the remark this answers is not in the document]
Comment (ViktarStarastsenka) [reply content/ri/release-notes/v2.10.0.md:14]: I added the description (it is basically to view and edit serialized PHP values as JSON)
the library we use for this does not provide many details, so can't reference it.

File: content/ri/release-notes/v2.10.0.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+- Enhanced overview for Redis databases with [OSS Cluster API](https://docs.redis.com/latest/rs/clusters/optimize/oss-cluster-api/) enabled to see memory and key allocation, as well as monitor database information per shards
```
[the remark this answers is not in the document]
Comment (ViktarStarastsenka) [reply content/ri/release-notes/v2.10.0.md:15]: @K-Jo , do you have any concerns about the new wording?

[the remark this answers is not in the document]
Comment (K-Jo) [reply content/ri/release-notes/v2.10.0.md:15]: remove the with now

**Review by ViktarStarastsenka — COMMENTED** [review]
fixed all the comments

File: content/ri/release-notes/v2.10.0.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+- [#1048](https://github.com/RedisInsight/RedisInsight/pull/1048) Ability to configure Workbench to not clear the Editor after commands have been run
+- [#1119](https://github.com/RedisInsight/RedisInsight/pull/1119) Move the Pipeline mode configuration for Workbench to the dedicated "Workbench" section in settings
+- [#1149](https://github.com/RedisInsight/RedisInsight/pull/1149) Group results in Workbench to see many results in one box to save space
+- [#1162](https://github.com/RedisInsight/RedisInsight/pull/1162) Add a link with User Survey
+- [#1037](https://github.com/RedisInsight/RedisInsight/pull/1037) Add tooltips for [Redis Streams](https://redis.io/docs/data-types/streams/) to display long fields
+- [#1202](https://github.com/RedisInsight/RedisInsight/pull/1202) Removed format validations from the admin username in the Redis Enterprise Cluster autodiscovery process
+- [#1179](https://github.com/RedisInsight/RedisInsight/pull/1179) Updated a message when there are no keys in Browser/Tree view to provide guidances on how to quickly add test data to Redis database
```
Comment (nermiller) [inline content/ri/release-notes/v2.10.0.md:31]: What is the most important user benefit resulting from this change?

File: content/ri/release-notes/v2.10.0.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+**Bugs**
+- [#1180](https://github.com/RedisInsight/RedisInsight/pull/1180) Truncate TTL in minutes
+- [#1197](https://github.com/RedisInsight/RedisInsight/pull/1197) Fix a case when Workbench is not available due to the failed encryption
+- [#1176](https://github.com/RedisInsight/RedisInsight/pull/1176) Save the refresh value in Browser/Tree view
+- [#1101](https://github.com/RedisInsight/RedisInsight/pull/1101) Fix an issue when key names are not displayed
```
Comment (nermiller) [inline content/ri/release-notes/v2.10.0.md:37]: Enable key name display?

**Review by nermiller — CHANGES_REQUESTED** [review]
Hi, @ViktarStarastsenka @K-Jo, I left some questions and comments.

File: content/ri/release-notes/v2.10.0.md (hunk 9 above)
Comment (ViktarStarastsenka) [reply content/ri/release-notes/v2.10.0.md:31]: let's just remove it

File: content/ri/release-notes/v2.10.0.md (hunk 10 above)
Comment (ViktarStarastsenka) [reply content/ri/release-notes/v2.10.0.md:37]: it was a very rare case when key names were not displayed, let's leave it as is?

File: content/ri/release-notes/v2.10.0.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+- [#1119](https://github.com/RedisInsight/RedisInsight/pull/1119) Move the Pipeline mode configuration for Workbench to the dedicated "Workbench" section in settings
+- [#1149](https://github.com/RedisInsight/RedisInsight/pull/1149) Group results in Workbench to see many results in one box to save space
+- [#1162](https://github.com/RedisInsight/RedisInsight/pull/1162) Add a link with User Survey
+- [#1037](https://github.com/RedisInsight/RedisInsight/pull/1037) Add tooltips for [Redis Streams](https://redis.io/docs/data-types/streams/) to display long fields
+- [#1202](https://github.com/RedisInsight/RedisInsight/pull/1202) Removed format validations from the admin username in the Redis Enterprise Cluster autodiscovery process
+- [#1179](https://github.com/RedisInsight/RedisInsight/pull/1179) Updated a message when there are no keys in Browser/Tree view to provide guidances on how to quickly add test data to Redis database
+
+**Bugs**
+- [#1180](https://github.com/RedisInsight/RedisInsight/pull/1180) Truncate TTL in minutes
+- [#1197](https://github.com/RedisInsight/RedisInsight/pull/1197) Fix a case when Workbench is not available due to the failed encryption
```
[the remark this answers is not in the document]
Comment (ViktarStarastsenka) [reply content/ri/release-notes/v2.10.0.md:35]: changed the wording here

File: content/ri/release-notes/v2.10.0.md (hunk 10 above)
Comment (nermiller) [reply content/ri/release-notes/v2.10.0.md]: sounds good to me

**Comment by nermiller** [comment]
LGTM!

**Pull request merged**